matrix_client/crypto/one_time_keys.py

```diff
@@ -0,0 +1,33 @@
+class OneTimeKeysManager(object):
+    """Handles one-time keys accounting for an OlmDevice."""
+
+    def __init__(self, target_keys_number, signed_keys_proportion):
+        self.target_counts = {
+            'signed_curve25519': int(round(signed_keys_proportion * target_keys_number)),
+            'curve25519': int(round((1 - signed_keys_proportion) * target_keys_number)),
+        }
+        self._counts = {}
+        self.to_upload = {}
+
+    @property
+    def counts(self):
```

Comment:
More descriptive name here please or brief docstring (what is this count of?).

Reply:
Changed to `key_counts`.

Reply:
Sorry. Should've been more explicit. Was looking for something like `server_counts` or even `server_key_counts`.